ballerina-shell/modules/shell-core/src/test/java/io/ballerina/shell/test/unit/EvaluatorMiscTest.java

```diff
@@ -68,9 +68,9 @@ public void testEvaluatorImportList() throws BallerinaShellException {
         evaluator.evaluate("import ballerina/lang.'float as prefix2");
         Assert.assertEquals(new HashSet<>(evaluator.availableImports()),
                 Set.of(
-                        "('java) import ballerina/'jballerina.'java as 'java;",
-                        "('prefix) import ballerina/'lang.'int as 'prefix;",
-                        "('prefix2) import ballerina/'lang.'float as 'prefix2;"
+                        "('java) import ballerina/jballerina.'java as 'java;",
```

Comment:
Is `java` a keyword? 🤔

Reply:
No, `java` is not a keyword. In the above case if we use `import ballerina/jballerina.java` output will be `(java) import ballerina/jballerina.java as java;`.

Reply:
Ah I see. Is there an explicit quote in the above test case?

Reply:
yes, I have updated the above test case without a quote in 214b385683a0cce12446b123aa7f2555f56d3326

Reply:
No, it's fine if that's how it's in the input of the test case.